.github/workflows/bench.yml

```diff
@@ -0,0 +1,62 @@
+name: Benchmark
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: ['opened', 'reopened', 'synchronize']
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.sha }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+env:
+  CI: 1
+  CARGO_INCREMENTAL: 0
+  # For faster CI
+  RUST_LOG: 'off'
+  # https://github.com/actions/setup-node/issues/899#issuecomment-1819151595
+  SKIP_YARN_COREPACK_CHECK: 1
```

Comment:
We should consider using `build_reusable.yml` to handle stuff like this, but that's not a blocker and can be a longer-term follow-up. https://github.com/search?q=repo%3Avercel%2Fnext.js%20build_reusable&type=code

Reply:
Wow! I didn't know that such thing is possible in GH Actions